agenthub/codeact_agent/codeact_agent.py

```diff
@@ -32,24 +27,11 @@
     JupyterRequirement,
     PluginRequirement,
 )
-
-ENABLE_GITHUB = True
-
-
-# FIXME: We can tweak these two settings to create MicroAgents specialized toward different area
-def get_system_message() -> str:
-    if ENABLE_GITHUB:
-        return f'{SYSTEM_PREFIX}\n{GITHUB_MESSAGE}\n\n{COMMAND_DOCS}\n\n{SYSTEM_SUFFIX}'
-    else:
-        return f'{SYSTEM_PREFIX}\n\n{COMMAND_DOCS}\n\n{SYSTEM_SUFFIX}'
-
-
-def get_in_context_example() -> str:
-    return EXAMPLES
+from opendevin.utils.prompt import PromptManager
 
 
 class CodeActAgent(Agent):
-    VERSION = '1.8'
+    VERSION = '1.9'
```

Comment:
Just to note, it's not obvious to me from the diff (looking on GitHub) that the prompt as sent to the llm really did change. But 1.9 is ok anyway, it's probably overdue.

Reply:
OK, it did change, though not a lot.